tools/labs/templates/networking/5-udp-sock/udp_sock.c

```diff
@@ -114,15 +114,13 @@ int __init my_udp_sock_init(void)
 	return 0;
 
 out_release:
-	/* TODO 1/0: cleanup socket */
```

Comment:
This should be:
`/* TODO 1/1: cleanup socket */`

I believe we should delete the sock_release() call for the student-facing skeleton, because in the task students have to open and release the socket.

Reply:
I've solved this in the newest commit.